host_core/lib/host_core/application.ex

```diff
@@ -119,7 +126,8 @@ defmodule HostCore.Application do
            ]
          }},
         id: :latticectl_consumer_supervisor
-      )
+      ),
+      {HostCore.Host, config}
```

Comment:
Were you able to do any testing as to if this affects the ability for the host to query cache, or does this effectively break the cache query entirely?

Reply:
I'm reasonably convinced this is going to break the cache. I can see the host answering this query. This problem will go away when we switch to the JetStream-based cache since we won't be asking hosts to answer the query, we'll be subscribing to the ephemeral consumers.